NEWS.md

```diff
@@ -1,3 +1,16 @@
+## 2.0.0 - Released
+
+This is a maintenance release focused on keeping dependencies up to date.  The major version change is due to the new permission requirements on APIs which were previously unrestricted.
+ 
+[Full Changelog](https://github.com/folio-org/mod-login-saml/compare/v1.3.0...v2.0.0)
+ 
+### Stories
+ * [MODLOGSAML-64](https://issues.folio.org/browse/MODLOGSAML-64) - Upgrade to RMB v30
+ * [MODLOGSAML-60](https://issues.folio.org/browse/MODLOGSAML-60) - Securing APIs by default
+
+### Bugs
+ * [MODLOGSAML-55](https://issues.folio.org/browse/MODLOGSAML-55) - org.pac4j:pac4j-saml vulnerability found in pom.xml on Nov 6, 2019
```

Comment:
Should this be mentioned here? It is a duplicate of MODLOGSAML-50 and that one is still open (no work was done)

Reply:
oh!  Shame on me...  I didn't do my homework.  I saw it was closed and marked as fixed in this version in JIRA.  I didn't notice it was closed with reason: duplicate.  You're right it doesn't belong here.